analyzers/tests/SonarAnalyzer.UnitTest/TestCases/GetTypeWithIsAssignableFrom.cs

```diff
@@ -92,6 +92,33 @@ public void Go(Repro_3605 value)
         }
     }
 
+    // https://github.com/SonarSource/sonar-dotnet/issues/6616
+    public class Repro_6616
+    {
+        public void IsInstanceOfType(object obj)
+        {
+            _ = typeof(ISet<int>).IsInstanceOfType(obj); // Noncompliant, bounded generic type
+            _ = typeof(ISet<>).IsInstanceOfType(obj); // Compliant, unbonded generic type
+            _ = typeof(IDictionary<int, int>).IsInstanceOfType(obj); // Noncompliant, bounded generic type
+            _ = typeof(IDictionary<,>).IsInstanceOfType(obj); // Compliant, unbonded generic type
```

Comment:
These comments would be way more readable when aligned on the same tab ditance

Reply:
Same below

Reply:
I have tried to line those comments up, but I bumped into the following issue by the code fix test `GetTypeWithIsAssignableFrom_CodeFix`: `Expected ActualCodeWithReplacedComments().ToUnixLineEndings() to be X  with a length of 5756 because VerifyWhileDocumentChanges updates the document until all issues are fixed, even if the fix itself creates a new issue again. Language: CSharp7, but Y has a length of 5819, differs near "   " (index 3683)`, where the only differences are in whitespaces:

![image](https://user-images.githubusercontent.com/2676742/215771146-6207dfe9-7731-4efd-8b53-515ad8930dcd.png)

I think that the test just preserves whitespaces when replacing `// Noncompliant` with `// Fixed` on a per-line basis, so lines end up misaligned in `GetTypeWithIsAssignableFrom.Fixed.cs` and `GetTypeWithIsAssignableFrom.Fixed.Batch.cs` (because fixes are of different length) and comparisons are sensitive to the difference in the number of whitespaces.

So we basically:
1. either have comments lined up in the main test-case file, and scrambled in `Fixed.cs` and `Fixed.Batch.cs`;
2. or have comments starting 1 chars from the `;` (end of instruction), on the three files.

While not optimal, I think option 2 is more consistent and makes `Fixed.cs` and `Fixed.Batch.cs` easier to edit.

If you still prefer option 1, I can make the change.

Reply:
Basically the reasoning is that now, it's not readable anywhere. After, it will be readable at least on the main source. While we don't care too much about the comments of the fixed part.